examples/liquid.ini

```diff
@@ -129,7 +129,7 @@ template = templates/zipkin.nomad
 
 # Add the testdata collection. Remove if not needed.
 # Be sure to clone it first:
-#     git clone https://github.com/hoover/testdata collections/testdata
+#     git clone https://github.com/liquidinvestigations/testdata collections/testdata
```

Comment:
And again, `hoover-testdata`

Reply:
I've already updated everything to use this name. There's no confusion, so I'll leave it like it is.